privacy_evaluator/utils/trainer.py

```diff
@@ -146,13 +164,14 @@ def _trainer_torch(
     class_encoding = {class_id: i for i, (class_id, _) in enumerate(size_dict.items())}
 
     # start training
-    for _ in range(num_epochs):
+    if verbose==2:
+        print("Training PyTorch model in ", num_epochs, "epochs.")
```

Comment:
missing space in front of "epochs." ?

Reply:
There's space after "in". But "print(a, b)" automatically adds a space between a and b (so technically there's a space too much in the above code :D)